### PR TITLE
Design tokens <> Tailwind

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

- workflow added for importing tokens
  -  packages/ui new scripts added for copying tokens
  -  packages/ui new scripts added for composing tokens and generating css using style-dictionary
  - scope to add more types of tokens, base tokens, typography etc.. but for now commented out
- all /apps and studio app updated tailwind config
  - can dictate app level config for tailwind which deep merges with baseline tailwind config for all apps
- updated storybook to v7
- removed all the old 'custom tailwind classes'. replaced with 'scale' based ones for now.

new classes are like:

```jsx
<>
  /* 
  * Body based colors use `-body` as the color indentifier
  */
<body className="bg-body border-body">
  /* 
  * Panels using 'default' classes
  */
  <div className="bg border p-3">I am a panel<div>
</body>
</>
```

## caveats

- we use mix() function in figma tokens which means, color mixes are done figma side for now.
  - there is no equivalent for this in CSS apart from `color-mix` but it's not widely supported in firefox yet
  - we set raw values for any color modifiers for now, but in future, we could handle the mixes in css variables.
  - if we want to support a custom user theming feature later, we will need to use JS to construct all the different shades.

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by kiwicopple — APPROVED** [review]

**Comment by kiwicopple** [comment]
@MildTomato approved, let's merge if possible so that it doesn't gather dust/conflicts

**Comment by MildTomato** [comment]
will fix merge conflicts and merge

**Comment by MildTomato** [comment]
waiting on another set of eyes from @supabase/frontend

File: studio/tailwind.config-old.js
Comment (alaister) [inline studio/tailwind.config-old.js:1]: Any reason we still need this file?

**Review by alaister — APPROVED** [review]
LGTM! Just the `package-lock.json` merge conflict to fix.

**Comment by MildTomato** [comment]
Got a new token naming convention coming through. will update with new changes.

File: studio/tailwind.config-old.js
Comment (MildTomato) [reply studio/tailwind.config-old.js:1]: I'll be getting rid of this and old files. i was just experimenting.

**Comment by MildTomato** [comment]
converted to draft to avoid accidental merge

**Pull request merged**
